pennylane/transforms/decompose.py

```diff
@@ -60,7 +61,53 @@ def _operator_decomposition_gen(
             )
 
 
-@transform
+def _decompose_plxpr_transform(
+    self, primitive, tracers, params, targs, tkwargs, state=None
```

Comment:
```suggestion
    primitive, tracers, params, targs, tkwargs, state=None
```

I was trying to replicate this example and it took me some time to figure out that this was the main issue XD